### PR TITLE
Fix/force nipt upload

### PR DESCRIPTION
**Pull request by karlnyr**
## Description

Forcing upload did not work as it was stopped down the line by daughter functions

### Fixed

- Force flag is properly propagated into daughter functions

### How to prepare for test
- [ ] ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] paxa the environment: `paxa`
- [ ] install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh fix/force-nipt-upload`

### How to test
- [ ]

### Expected test outcome
- [ ] check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in ...
- [ ] Deploy this branch
- [ ] Inform to @Clinical-Genomics/production

### CODE REVIEW

File: cg/cli/upload/nipt/base.py [hunk 1]
```diff
@@ -38,13 +38,13 @@ def nipt_upload_case(context: click.Context, case_id: Optional[str], dry_run: bo
 
     nipt_upload_api: NiptUploadAPI = NiptUploadAPI(context.obj)
     nipt_upload_api.set_dry_run(dry_run=dry_run)
-
+    print(force)
```
Comment (henrikstranneheim) [inline cg/cli/upload/nipt/base.py:41]:
```suggestion
```

**Review by henrikstranneheim — APPROVED** [review]
👍

**Review by barrystokman — APPROVED** [review]
🍴

**Comment by karlnyr** [comment]
Installed in prod:
image

Proof of function in prod:
Screen Shot 2022-02-15 at 14 31 50

**Pull request merged**